src/NodaTime/Utility/FileLoader.cs

```diff
@@ -0,0 +1,35 @@
+﻿// Copyright 2011 The Noda Time Authors. All rights reserved.
+// Use of this source code is governed by the Apache License 2.0,
+// as found in the LICENSE.txt file.
+
+using System.IO;
+using System.Net.Http;
+
+namespace NodaTime.Utility
+{
+    /// <summary>
+    /// Provides method to help with loading file or URL.
+    /// </summary>
+    public static class FileLoader
```

Comment:
I'm reluctant to put this into NodaTime itself. It's not used within NodaTime - you've only included it here because it's a common dependency, right?

Reply:
Yes. Where else would you include it?